packages/spark/settings/_settings.scss

```diff
@@ -443,15 +443,17 @@ $sprk-text-input-font-weight: 300 !default;
 $sprk-text-input-height: 48px !default;
```

Comment:
@guern1kn I don't think you should change anything right now, but I think we need to discuss our variable naming and inheritance strategy as a team. There's so many things here that should probably just inherit form other vars and not just be their stand values.

Reply:
Sounds like a good meta topic!

Reply:
This is probably part of a bigger discussion about Design Tokens
https://github.com/sparkdesignsystem/spark-design-system/issues/1120